dotCMS/src/main/java/com/dotmarketing/servlets/InitServlet.java

```diff
@@ -260,7 +255,9 @@ public void init(ServletConfig config) throws ServletException {
         catch(Exception e){
             Logger.warn(this.getClass(), "Unable to record startup time :" + e);
         }
-
+        // Starting the re-indexation thread
+        ReindexThread.startThread(Config.getIntProperty("REINDEX_THREAD_SLEEP", 500),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 500 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)